python/grass/temporal/core.py

```diff
@@ -1066,8 +1069,8 @@ def _create_tgis_metadata_table(content, dbif=None):
 
 
 class SQLDatabaseInterfaceConnection(object):
-    def __init__(self):
-        self.tgis_mapsets = get_available_temporal_mapsets()
+    def __init__(self, only_current_mapset=False):
+        self.tgis_mapsets = get_available_temporal_mapsets(only_current_mapset)
```

Comment:
Just a stylistic note: There is a value in writing this like `only_current_mapset=only_current_mapset` [because it is a boolean flag](https://ivergara.github.io/boolean-arguments-to-functions-in-python.html).

The best practice would be probably to enforce it with a [keyword-only argument](https://peps.python.org/pep-3102/) syntax (`, *,`) both in the function and here.  We were not using it, so far, we needed more the new positional only arguments, but keyword-only arguments are in Python 3.0 already and I just used in [PR 2367](https://github.com/OSGeo/grass/pull/2367/files#diff-4815fb3068aa191296cfc3394d441aeb6ae6af2c339930bca4c23544c254bb1dR2041).

Unless you document it as not part of the API (or in the future use a positional-only argument), the name `only_current_mapset` is part of the API anyway.

Reply:
OK, done.